### ISSUE TITLE
Missing dependencies for `pip install huxley`

### ISSUE THREAD

**Issue by ericclemmons** [facebookarchive/huxley#22]
`pip install huxley` failed, so I had to run the following _individually_:

```
$ pip install jsonpickle
$ pip install plac
$ pip install selenium
$ pip install pil
```

Finally, I could run:

```
$ pip install huxley
```

Want me to add this to the README?

**Comment by ericclemmons** [comment]
Yikes, and it gets worse.  Apparently `pip install jsonpickle` installs `0.6.1`, whereas huxley needs `0.4.0`:

```
$ ⮀huxley
Traceback (most recent call last):
  File "/usr/local/bin/huxley", line 5, in <module>
    from pkg_resources import load_entry_point
  File "/usr/local/Cellar/python/2.7.5/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/distribute-0.6.45-py2.7.egg/pkg_resources.py", line 2854, in <module>
    parse_requirements(__requires__), Environment()
  File "/usr/local/Cellar/python/2.7.5/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/distribute-0.6.45-py2.7.egg/pkg_resources.py", line 594, in resolve
    raise DistributionNotFound(req)
pkg_resources.DistributionNotFound: jsonpickle==0.4.0
```

**Comment by petehunt** [comment]
Ew, seems like I messed up the setup.py file. Will definitely try to get this one fixed, thanks!

**Comment by ericclemmons** [comment]
Geeze @petehunt!  Never thought I'd expect that from _you_.  You're right, that is ewwww...

</x-good-mood>

**Comment by plasticine** [comment]
Any ETA on a fix for this? Not being able to `pip install` is sad :(

**Comment by klamping** [comment]
FYI, for anyone else interested in the steps to install with the dependencies the way they are, here's how I did it:
1. First, you need to have Opera installed before all of this. Do that here: http://www.opera.com/computer/
2. Install jsonpickle v0.4.0 (i had to use sudo, you may not)
   
   `$ sudo pip install jsonpickle==0.4.0`
3. Install the remaining dependencies (again, i had to use sudo, you may not)
   
   `$ sudo pip install plac`
   `$ sudo pip install selenium`
   `$ sudo pip install pil`
4. Then you can install huxley itself:
   `$ sudo pip install huxley`

**Comment by petehunt** [comment]
I'll try to cut a new release this week to fix this.

Sent from my iPhone

On Oct 6, 2013, at 3:38 PM, "Kevin Lamping" <notifications@github.com<mailto:notifications@github.com>> wrote:

FYI, for anyone else interested in the steps to install with the dependencies the way they are, here's how I did it:
1.  First, you need to have Opera installed before all of this. Do that here: http://www.opera.com/computer/https://urldefense.proofpoint.com/v1/url?u=http://www.opera.com/computer/&k=ZVNjlDMF0FElm4dQtryO4A%3D%3D%0A&r=qYx6qLphxKhA5vHBqr9vuw%3D%3D%0A&m=XWNZ24MvzfBT0lAhk3D17tqvFcThJESXcfTkUXmpVQY%3D%0A&s=097519243086065b3da1e9a08e8f6cc23c853e642e77bc4519887dbfc2fc2caf
2.  Install jsonpickle v0.4.0 (i had to use sudo, you may not)

$ sudo pip install jsonpickle==0.4.0
1.  Install the remaining dependencies (again, i had to use sudo, you may not)

$ sudo pip install plac
$ sudo pip install selenium
$ sudo pip install pil
1.  Then you can install huxley itself:
   $ sudo pip install huxley

**Comment by petehunt** [comment]
Fixed in 0.4. Thanks!
